src/rules/tsUtils.ts

```diff
@@ -59,13 +77,10 @@ export const isFunction = (node: TSESTree.Node): node is FunctionExpression =>
 /* istanbul ignore next */
 export const isHook = (
   node: TSESTree.CallExpression,
-): node is JestFunctionCallExpression<HookName> => {
+): node is JestFunctionCallExpressionWithIdentifierCallee<HookName> => {
   return (
-    (node.callee.type === AST_NODE_TYPES.Identifier &&
-      node.callee.name in HookName) ||
-    (node.callee.type === AST_NODE_TYPES.MemberExpression &&
-      node.callee.object.type === AST_NODE_TYPES.Identifier &&
-      node.callee.object.name in HookName)
+    node.callee.type === AST_NODE_TYPES.Identifier &&
+    node.callee.name in HookName
```

Comment:
I've changed this b/c jest doesn't expect hooks to expose a method property, and this way the two rules that use them don't need to be changed :D

Reply:
I don't follow - you can do `describe.skip` (etc). Or do I misunderstand your comment?

Reply:
But `describe` isn't a hook - the `HookName` enum is for `beforeAll`, `beforeEach`, `afterEach` & `afterAll`.

That's why it's the only one I made this change to - the others I left as it b/c in the JS version they have member calls (i.e `describe.skip`)

Reply:
hah, good point! There are indeed no identifiers on _hooks_